### ISSUE TITLE
Mnemonics with Parameters flagged as error

### ISSUE THREAD

**Issue by StephanWald** [BBx-Kitchen/bbj-language-server#39]
This is valid code:

```
? 'FILESAVE'("save file","C:/","test"",".txt","Text files"+$0a$+"*.txt")
? 'WINDOW'(10,10,20,10,"Hello")
```

Many of the Mnemonics listed here do have parameters:

https://documentation.basis.cloud/BASISHelp/WebHelp/mnemonic/Alphabetical_Mnemonics.htm

**Comment by dhuebner** [comment]
@StephanWald 
Is the `""` at the end of `,"test""` a typo?

**Comment by StephanWald** [comment]
Yes it's a typo sorry

**Comment by dhuebner** [comment]
Fixed with #58
